packages/react-select/src/Select.js

```diff
@@ -608,7 +690,7 @@ export default class Select extends Component<Props, State> {
     });
     this.announceAriaLiveContext({
       event: 'menu',
-      context: { isDisabled: isOptionDisabled(options[nextFocus]) },
+      context: { isDisabled: this.isOptionDisabled(options[nextFocus], selectValue) },
```

Comment:
I assumed that this was a bug. It was calling the builtin `isOptionDisabled`, instead of the `this.isOptionDisabled` which calls `this.props.isOptionDisabled`.